src/fe/index.js

```diff
@@ -29,6 +29,26 @@ function getStatus() {
     );
     // set the response span text to the response text content
     // don't forget to get the result of the promise
+    response.json().then(function(obj) {
+      responseSpan.textContent = obj.status;
```

Comment:
obj.status doesn't actually exist i don't think - this should just be 

```javascript
responseSpan.textContent = obj
```

I think

P.S. this smells of copilot haha

Reply:
Done - I think anything I've added to the FE has been either entirely or almost entirely chatGPT / co-pilot hahah